src/App.jsx

```diff
@@ -1,7 +1,78 @@
+import { nanoid } from "nanoid";
+import { useEffect, useState } from "react";
+import Confetti from "react-confetti";
+import Dice from "./components/Dice";
+
+const generateRandomDiceValue = () => {
+  return Math.ceil(Math.random() * 6);
+};
+
+const generateDiceValues = () => {
+  return Array.from(Array(10).keys()).map(() => ({
+    id: nanoid(),
+    value: generateRandomDiceValue(),
+    isHeld: false,
+  }));
+};
+
 export default function App() {
```

Comment:
since you're using arrow functions, you can also make this one

Reply:
The system here was that I was using the function declarations for components and the arrow function for the rest of the functions. But to avoid confusion I can change them to be arrow 

Thanks for the catch